classes/session/SessionManager.inc.php

```diff
@@ -215,7 +221,14 @@ function gc($maxlifetime) {
 	 * @return boolean
 	 */
 	function updateSessionCookie($sessionId = false, $expireTime = 0) {
-		$domain = ini_get('session.cookie_domain');
```

Comment:
Could you describe this change in a little more detail? I would expect, as long as we set the domain correctly upon session creation, and use `ini_set` to store it, then `ini_get` should always be reliable.

Reply:
yes, I have a potential solution in the discussion above